File: week2/Bit++/bit++.py
"""
Problems: Bit++
Link: https://codeforces.com/contest/282/problem/A
Tags: implementation
Complexity: O(n)
"""
n = int(input())
x = 0
for _ in range(n):
    oper = input()
    if oper[1] == '+':
        x += 1
    else:
        x -= 1
print(x)
# Solved by: Muaath Alqarni

File: week2/Bit++/bit++.cpp
/*
Problem: Bit++
Link: https://codeforces.com/contest/282/problem/A
Tags: implementation
Complexity: O(n)
*/
#include <bits/stdc++.h>
#define ll long long
using namespace std;

int n, x = 0;

int main()
{
    ios::sync_with_stdio(0), cin.tie(0), cout.tie(0);
    cin >> n;
    while (n--) {
        string str;
        cin >> str;
        if (str[1] == '+')
            x++;
        else
            x--;
    }
    cout << x;
}
// Solved by: Muaath Alqarni